snap/hooks/install

```diff
@@ -19,11 +19,10 @@ EOF
 	exit 1
 fi
 
-# Create directories required by FRR
-frr_etc="$SNAP_COMMON/data/frr/etc/"
-mkdir -p $frr_etc
-mkdir -p "$SNAP_COMMON/data/frr/tmp"
-mkdir -p "$SNAP_COMMON/run/frr"
+# Create directories required by Bird
+bird_data_dir="$SNAP_COMMON/data/bird"
+mkdir -p "$bird_data_dir"
+mkdir -p "$SNAP_COMMON/run/bird"
```

Comment:
The other directories in `$SNAP_COMMON/data` and `$SNAP_COMMON/run` have `0o700` permissions, while these are created with `0o755`.

I don't see the other scripts being explicit about it, so it may be install hook is ran with a different umask.

In any case, let's be consistent and tighten the permissions for these directories.